packages/web-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,68 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  colorCompoundBrandForeground1,
+  colorNeutralForeground1,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeAccessible,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+} from '../theme/design-tokens.js';
+
+/** RadioGroup styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --control-border-color: ${colorNeutralStrokeAccessible};
+    --checked-indicator-background-color: ${colorCompoundBrandForeground1};
+    --state-color: ${colorNeutralForeground3};
+    align-items: flex-start;
+    flex-direction: column;
+    row-gap: ${spacingVerticalS};
+  }
+  :host([disabled]) {
+    --control-border-color: ${colorNeutralForegroundDisabled};
+    --checked-indicator-background-color: ${colorNeutralForegroundDisabled};
```

Comment:
since these are for radio I'd move the definitions for these there. Then simply reassign them in your disabled selector.

so

```
host([disabled]) ::slotted([role='radio']) {
    --control-border-color: ${colorNeutralForegroundDisabled};
    --checked-indicator-background-color: ${colorNeutralForegroundDisabled};
    --state-color: ${colorNeutralForegroundDisabled};
}
```